frontend/src/pages/SignupPage/index.js

```diff
@@ -80,6 +80,18 @@ const SignupPage = () => {
       }
     };
 
+  const validatesChecked = (id) => {
+    if (id === "other") {
+      return state.otherChecked;
+    } else if (id === "jobs") {
+      return state.consent;
+    } else if (id === "terms") {
```

Comment:
What about extracting these to constants?

Reply:
Okay seems good, will do that in the afternoon after I finish my current task :smile: